task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -6,12 +6,24 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Enter the first number"); 
+            Console.WriteLine("Enter the first number");
             int firstInputNumber = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter the second number");
             int secondInputNumber = int.Parse(Console.ReadLine());
+            int min = 0;
+            int max = 0;
 
-            for (int i = firstInputNumber; i <= secondInputNumber; i++)
+            if (firstInputNumber < secondInputNumber || firstInputNumber == secondInputNumber)
+            {
+                min = firstInputNumber;
+                max = secondInputNumber;
+            }
+            else if (firstInputNumber > secondInputNumber)
```

Comment:
Why do you explicitly check that first is more then second? I guess `else `would be enough here.